### ISSUE TITLE
Polybar no longer print font awesome

### ISSUE THREAD

**Issue by eoli3n** [jaagr/polybar#941]
Hi,
From ``ttf-font-awesome-4.7.0-5-any.pkg.tar.xz`` to ``ttf-font-awesome-5.0.2-1-any.pkg.tar.xz`` on Archlinux breaks polybar which doesn't display font anymore.

Last polybar-git version.

In ``.config/polybar/config``
```
font-3 = FontAwesome:pixelsize=14;2
```
When launching
```
warn: Dropping unmatched character  (U+f0ac)
warn: Dropping unmatched character  (U+f054)
warn: Dropping unmatched character  (U+f185)
warn: Dropping unmatched character  (U+f028)
warn: Dropping unmatched character  (U+f021)
warn: Dropping unmatched character  (U+f0ec)
warn: Dropping unmatched character  (U+f1eb)
warn: Dropping unmatched character  (U+f240)
```
As https://github.com/jaagr/polybar/issues/392 , tried to update font cache, didn't work.

FROM

![alt tag](https://github.com/eoli3n/dotfiles/blob/master/screenshots/i3gaps.png)

TO 

![alt tag](https://ptpb.pw/vW_k.png)

**Comment by patrick96** [comment]
duplicate of #924 
TL;DR you'll need to update your font specifiers and also possibly some of your icons, since FA5 may use other icon codes

**Comment by eoli3n** [comment]
Oh, sorry, didn't search ... aur ttf-font-awesome-4 fixed it
